doc/classes/ClassDB.xml

```diff
@@ -192,6 +192,15 @@
 				Returns whether this [param class] is enabled or not.
 			</description>
 		</method>
+		<method name="is_class_enum_bitfield" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="class" type="StringName" />
+			<param index="1" name="enum" type="StringName" />
+			<param index="2" name="no_inheritance" type="bool" default="false" />
+			<description>
+				Returns whether the [param enum] is defined as a bitfield in [param class] or its ancestry classes.
```

Comment:
"ancestry" sounds weird to me here. Maybe:
```suggestion
				Returns whether the [param enum] is defined as a bitfield in [param class] or its ancestor classes.
```

Reply:
```suggestion
				Returns whether the [param enum] is defined as a bitfield in [param class] (or its ancestry if [param no_inheritance] is [code]false[/code]).
```

Reply:
I believe I took this from the other methods in ClassDB, should we adjust the other descriptions here or should be do that in a separate PR entirely?

Reply:
Ah, if it's in other places, then a separate PR probably makes sense

Reply:
Perfect, updated the newly added method only, thanks to you both 👍